tools/bionano/bionano_scaffold.xml

```diff
@@ -420,22 +420,26 @@
         -->
     </inputs>
     <outputs>
-        <data name="ngs_contigs" format="fasta" from_work_dir="total_contigs_raw.fasta" label="${tool.name} on ${on_string}: NGScontigs">
```

Comment:
@Delphine-L is that everything that you need?

Reply:
Yes, I think so, those are the required files that she told me.